plugins/module_utils/dnac.py

```diff
@@ -331,7 +332,14 @@ def check_task_response_status(self, response, validation_string, data=False):
             return self
 
         task_id = response.get("taskId")
+        start_time = time.time()
         while True:
+            end_time = time.time()
+            if (end_time - start_time) >= self.max_timeout:
+                self.log("Max timeout of {0} sec has reached for the task id '{1}' for the event and unexpected "
```

Comment:
Can we spilt this?
        self.log("Max timeout of {0} seconds reached for task ID '{1}'. Exiting the loop due to unexpected API status.".format(self.max_timeout, task_id), "WARNING")